Makefile

```diff
@@ -15,11 +22,11 @@ requirements: virtualenv requirements.txt
 requirements_for_test: virtualenv requirements_for_test.txt
 	${VIRTUALENV_ROOT}/bin/pip install -r requirements_for_test.txt
 
-npm_install:
+npm_install: package.json
```

Comment:
What's the benefit of the dependency? As far as I understand `package.json` is always there and since `npm_install` is a phony step there are no timestamps to compare.